package/yast2-security.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Jun 12 16:27:53 UTC 2015 - ancor@suse.com
+
+- When checking services, systemd aliases are now taken into
+  account (so, for example, rsyslog is accounted and syslog).
```

Comment:
Um, does not sound clear to me, maybe `accounted and syslog` should be `accounted as syslog` instead??

And BNC or FATE missing (this will be rejected in SLE12-SP1).